### PR TITLE
Friendlier syntax

### PR DESCRIPTION
**Pull request by 0x787af25e**
The user can now type shorter commands, for example h in place of help.

Added some test cases too.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -16,7 +18,7 @@
  */
 public class AddCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final List<String> COMMAND_WORD = List.of(new String[]{"add", "a"});
```
Comment (jefrai) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:21]:
```suggestion
    public static final List<String> COMMAND_WORDS = List.of(new String[]{"add", "a"});
```

Perhaps change all occurrences of `COMMAND_WORD` to plural `COMMAND_WORDS` to reflect the new usage?

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -16,7 +18,7 @@
...
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
```
Comment (jefrai) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:23]: Does this line still produce the right output? Could either stick to the original text, or if anyone has ideas could also somehow introduce the new shortcut to the user

**Review by jefrai — CHANGES_REQUESTED** [review]
Seems like a simple & clean addition - clear to merge after resolving:~

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 2 above)
Comment (0x787af25e) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:23]: Yeah this works fine, so after this I will allow users to define which shortcut they want

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (0x787af25e) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:21]: Done

**Pull request merged**